mod/module.go

```diff
@@ -48,7 +51,12 @@ func (m *Modules) Len() int {
 	return len(*m)
 }
 
+var configMutex = &sync.RWMutex{}
+
```

Comment:
I'm not sure this is a good way to use mutex, we should attach this to the structs what will have some write operations.